auth.go

```diff
@@ -28,14 +28,18 @@ const (
 	authenticatorOkta            = "OKTA"
 )
 
-// platform consists of compiler, OS and architecture type in string
-var platform = fmt.Sprintf("%v-%v-%v", runtime.Compiler, runtime.GOOS, runtime.GOARCH)
+// platform consists of compiler and architecture type in string
+var platform = fmt.Sprintf("%v-%v", runtime.Compiler, runtime.GOARCH)
+
+// operatingSystem is the runtime operating system.
+var operatingSystem = runtime.GOOS
```

Comment:
Moved OS info from OS_VERSION to OS.

Currently no OS version can be obtained by API unless it runs a command, e.g., uname, cmd ver. I want to avoid to run an external command.